_data-prepper/monitoring.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Monitoring
+nav_order: 12
+---
+
+# Monitoring Data Prepper with Metrics
+
+Monitoring metrics in Data Prepper is instrumented using [Micrometer](https://micrometer.io/). There are two types of metrics: JVM and system metrics, and plugin metrics. [Prometheus](https://prometheus.io/) is used as the default metrics backend.
+
+## JVM and system metrics
+
+JVM and system metrics are runtime metrics used to monitor Data Prepper instances. They include metrics for classloaders, memory, garbage collection, threads, and others. For details, [JVM and system metrics](https://micrometer.io/docs/ref/jvm) for details. 
+
+### Naming
+
+JVM and system metrics in Data Prepper follow pre-defined names in Micrometer, such as `jvm_classes_loaded`, and `jvm_memory_used`. 
+
+### Serving
+
+By default, metrics are served from the **/metrics/sys** endpoint on the Data Prepper server. The format is a text Prometheus scrape. This port can be used for any frontend which accepts Prometheus metrics, such as [Grafana](https://prometheus.io/docs/visualization/grafana/). The configuration can be updated to serve metrics to other registries like CloudWatch which does not require or host the endpoint, but publishes the metrics directly to CloudWatch.
```

Comment:
Are the metrics always, without exception, published to CloudWatch? Or is CloudWatch the example throughout this statement? Some other registry does not require or host the endpoint but publishes the metrics directly?

Reply:
My understanding is that CloudWatch registry is one example but not the only one, and there're other registries available as well. 

> ...which does not require or host the endpoint, but publishes the metrics directly to CloudWatch.

This refers to CloudWatch registry only. If user configures Data Prepper to serve metrics to CloudWatch registry, it  publishes the metrics directly to CloudWatch.

Reply:
@oeyh Would it work if we rephrased to "This port can be used to serve metrics to other registries, like Amazon CloudWatch, which does not require or host the endpoint, and publishes the metrics directly to a CloudWatch frontend, such as [Grafana](https://prometheus.io/docs/visualization/grafana/)."?

Right now, it reads: "...and publishes the metrics directly to CloudWatch, such as Grafana" which sounds very ambiguous.

Reply:
@chenqi0805 Would it work if we rephrased to "This port can be used to serve metrics to other registries, like Amazon CloudWatch, which does not require or host the endpoint, and publishes the metrics directly to a CloudWatch frontend, such as [Grafana](https://prometheus.io/docs/visualization/grafana/)."?

Right now, it reads: "...and publishes the metrics directly to CloudWatch, such as Grafana" which sounds very ambiguous.

Reply:
@dlvenable Would welcome your feedback on this as well.

Reply:
I think the modified description is a bit misleading. Cloudwatch does not rely on any port or prometheus endpoint described here. Only Grafana frontend does. So the original description seems fine to me:


This port can be used for any frontend which accepts Prometheus metrics, such as [Grafana](https://prometheus.io/docs/visualization/grafana/). The configuration can be updated to serve metrics to other registries like CloudWatch which does not require or host the endpoint, but publishes the metrics directly to CloudWatch.